src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java

```diff
@@ -43,18 +47,127 @@ public void anonUserOnDesktopCanNotWriteNewPost() {
+  @Execute(asUser = User.USER)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void loggedInUserCanExpandPostEditorOnDesktop() {
+    PostsCreatorDesktop postsCreator = new PostsListPage().open().getPostCreatorDesktop();
+
+    postsCreator.click();
+
+    Assertion.assertTrue(postsCreator.isExpanded());
+    Assertion.assertFalse(postsCreator.isPostButtonActive());
+  }
+
+  /**
+   * This test covers all situations when post cannot be added (submit button is disabled).
+   *
+   * | Category | Title | Description |
+   * |          |     x |             |
+   * |          |     x |           x |
+   * |          |       |           x |
+   * |        x |       |             |
+   * |        x |     x |             |
+   *
+   * * x - means category was selected or text was added
+   */
+  @Test(groups = "discussions-loggedInUsersDesktopPosting")
+  @Execute(asUser = User.USER)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void userOnDesktopCannotSavePostWithoutCategoryAndDescription() {
+    PostsListPage postListPage = new PostsListPage().open();
+    PostsCreatorDesktop postsCreator = postListPage.getPostCreatorDesktop();
+
+    postsCreator.click()
+        .closeGuidelinesMessage();
+    Assertion.assertFalse(postsCreator.isPostButtonActive());
+
+    postsCreator.fillTitleWith(TEXT);
+    Assertion.assertFalse(postsCreator.isPostButtonActive(),
+        "User should not be able to add post with only title filled.");
+
+    postsCreator.fillDescriptionWith(TEXT);
+    Assertion.assertFalse(postsCreator.isPostButtonActive(),
+        "User should not be able to add post with title and description filled.");
+
+    postsCreator.clearTitle();
+    Assertion.assertFalse(postsCreator.isPostButtonActive(),
+        "User should not be able to add post with only description filled.");
+    postsCreator.clearDescription();
+
+    postsCreator.clickAddCategoryButton()
+        .findCategoryOnPosition(0)
+        .click();
+    Assertion.assertFalse(postsCreator.isPostButtonActive(),
+        "User should not be able to add post with only category selected.");
+
+    postsCreator.fillTitleWith(TEXT);
+    Assertion.assertFalse(postsCreator.isPostButtonActive(),
+        "User should not be able to add post with category selected and title filled.");
+
+  }
+
+  @Test(groups = "discussions-loggedInUsersDesktopPosting")
+  @Execute(asUser = User.USER)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void userOnDesktopCanAddPostWithoutTitle() {
+    final String description = createUniqueDescription();
+
+    PostsListPage postListPage = new PostsListPage().open();
+    PostsCreator postsCreator = postListPage.getPostCreatorDesktop();
+
+    final CategoryPill categoryPill = fillPostCategoryWith(postsCreator, description);
+
+    postsCreator.clickSubmitButton()
+                .waitForSpinnerToAppearAndDisappear();
```

Comment:
Is spinner always appearing ?
This test failed because it couldn't find the spinner:
http://jenkins:8080/job/test-environment-stability-check-one-execution/1/count=1,groups=$Group,label=webdriver/artifact/logs/log.html#toc1